CGPs/0009.md

```diff
@@ -0,0 +1,81 @@
+
+### Towards a more decentralized reserve
+The Celo community has expressed desire for the reserve to become more decentralized.  One key driver for this is the development of automated cross chain bridges that would allow reserve management to be automated.  Presently, the [Ren project](https://renproject.io/) provides a trustless bridge between Zcash and Ethereum.  Porting this to Celo would likely be a relatively light engineering lift.  A [privacy preserving trustless bridge](https://github.com/ZcashFoundation/zcash-pegzone) is also under construction between Zcash and Cosmos.  Other Zcash<->Ethereum bridges have been proposed by the organizations behind [tBTC](https://tbtc.network/) and [wBTC](https://wbtc.network/).  
+
+Funding for ongoing Zcash development is presently available from the [Zcash Foundation grants program](https://grants.zfnd.org/) and the forthcoming [Zcash Major Grants program](https://www.zfnd.org/blog/major-grants-review-committee-selection-process/).  It's likely that a proposal to build a trustless bridge between Zcash and Celo could be co-funded by sources from both the Zcash and Celo communities.
+
+### Cryptographic DNA
+The Zcash project was founded upon strong science, and the academic work of the Zcash [scientific founders](https://electriccoin.co/team/#scientists) has been foundational to various aspects of cryptocurrency.  Notably, [Eran Tromer](https://www.cs.tau.ac.il/~tromer/), who was one of the Zcash founding scientists, is also an advisor to the Celo project. Tromer has recently contributed to the development of the new [Plumo light client protocol](https://docs.zkproof.org/pages/standards/accepted-workshop3/proposal-plumo_celolightclient.pdf), which leverage zk-SNARKs to enable mobile devices to use Celo without downloading the entire blockchain.
+
+### About the author
+My name is [David Campbell](https://keybase.io/alchemydc), and I am responsible for operations at the ECC.  I also operate (personally) a [validator group](https://www.zanshindojo.org/) on the Celo network, and participated in the Great Celo Stake Off.  I also contributed a [significant refactor](https://github.com/celo-org/celo-monorepo/pull/3888) of the Terraform code published by cLabs to support running mainnet validators in GCP.  More information about my interest in cryptography and privacy and a disclosure statement is available [here](https://forum.zcashcommunity.com/t/dc-for-mgrc/36666).
+
+## Proposed Changes
+The present target allocation of the Celo reserve is as follows:
+* 50% CELO
+* 30% BTC
+* 15% ETH
+* 5% DAI
+
+This proposal advocates that the target allocation of the Celo reserve be modified as follows:
+* 50% CELO
+* 30% BTC
+* 10% ETH
+* 5% DAI
+* 5% ZEC
```

Comment:
I like the idea of diversifying the assets in the reserve further, specially with assets that have different features and could enable new use-case shall a bridge be created. 

As the primary role for the reserve is to provide collateral for cUSD, I'd think the community would be interested about the economics of Zcash, like what's the expected circulation supply in the near future and what factors are key to increase its value. I personally consider fundamental analysis for every asset wanted to be added to the reserve a must.

Is there some document that you can provide regarding this?

Reply:
Hi Martin,

Thanks for the feedback.

Zcash follows the same issuance curve as Bitcoin.  The first Zcash halvening is [coming up in November](https://www.nicehash.com/countdown/ZEC-halving-2020-11-17-12:00#!).

One particularly large factor to consider is that the Zcash community [recently decided](https://electriccoin.co/reaching-consensus/) to allocate 20% of issuance for the four years following the halvening to a development treasury.  Notably, the largest slice of this treasury (8% of issuance) is allocated to provide funding for third parties to develop and extend the Zcash platform and ecosystem.

Another factor to consider is that the ECC team continues to [develop and release innovative cryptography](https://www.coindesk.com/zcashs-halo-breakthrough-is-a-big-deal-not-just-for-cryptocurrencies) that solves problems that need to be solved for Zcash specifically, and also blockchains more generally.

Here are some links that provide additional perspective on Zcash for the Celo community:

[Placeholder Zcash Thesis](https://www.placeholder.vc/blog/2020/9/15/zcash-thesis)
[Zcash Analysis by BraveNewCoin](https://bravenewcoin.com/insights/zcash-price-analysis-transactions-per-day-continues-to-rise)
[Messari Profile](https://messari.io/asset/zcash/profile)

DC